site/gatsby-site/src/components/SimilaritySelector.js

```diff
@@ -87,6 +87,10 @@ const SimilaritySelector = ({
             btnProps.color = 'light';
           }
 
+          if (index === 1) {
+            btnProps.className = 'rounded-none';
+          }
+
```

Comment:
What is the purpose of this special condition?

Reply:
There is an issue with group buttons from flowbite where if you have more than 2 buttons, the ones in the middle sill apply a rounded border when they shouldn't. So in order for it to look 'nice' I had to manually remove the rounded borders.
<img width="317" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/4105289/f7f00a3a-3197-44b4-a711-faca994452db">

Reply:
Gotcha! thanks for the explanation